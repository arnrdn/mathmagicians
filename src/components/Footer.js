import React from 'react';

const Footer = () => (
  <footer>
    <p className="author">Made by Nana</p>
  </footer>
);

export default Footer;
