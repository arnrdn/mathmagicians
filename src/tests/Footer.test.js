import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

it('Match the Footer component snapshot', () => {
  const tree = renderer.create(<Footer />).toJSON();

  expect(tree).toMatchSnapshot();
});
