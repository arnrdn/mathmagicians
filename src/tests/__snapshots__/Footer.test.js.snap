// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the Footer component snapshot 1`] = `
<footer>
  <p
    className="fter author"
  >
    Made by
    <a
      href="https://github.com/rdnrn"
    >
       Nana
    </a>
  </p>
  <p
    className="fter"
  >
    <a
      href="https://icons8.com/icon/DchOf6erOUjJ/calculator"
    >
      Calculator icon by Icons8
    </a>
  </p>
</footer>
`;
